sedy/src/answerer.js

```diff
@@ -0,0 +1,13 @@
+export default (githubClient, parsedContent) => {
```

Comment:
Not fan of the nem, you should try something more specific, like "CodeReviewCommentAnswerer"